custom-images/.env

```diff
@@ -16,7 +16,7 @@ ID_HOST=id.dockerexamples.localhost
 SITECORE_DOCKER_REGISTRY=scr.sitecore.com/sxp/
 SITECORE_TOOLS_REGISTRY=scr.sitecore.com/tools/
 SITECORE_MODULE_REGISTRY=scr.sitecore.com/sxp/modules/
-SITECORE_VERSION=10.0.0-ltsc2019
+SITECORE_VERSION=10.0-ltsc2019
```

Comment:
Any risk not locking to specific patch (10.0.1)? I suppose it depends on the release/tagging strategy for this repo.

Reply:
Trying to keep as close to the container support package as possible here, and also cut down on the # of releases (in case there were a 10.0.2).

Reply:
Makes sense